client/src/components/map/layers/networks-layer/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { useMemo } from 'react';
+
+import { GeoJSONSourceSpecification } from 'maplibre-gl';
+import { Source, Layer, SymbolLayer } from 'react-map-gl/maplibre';
+
+import { LayerProps } from '@/types/layers';
+
+import { useMapNetworks } from '@/hooks/networks';
+
+export type NetworksLayerProps = LayerProps & {
+  beforeId?: string;
+};
+
+const getClusterSource = (source: GeoJSONSourceSpecification): GeoJSONSourceSpecification => ({
+  type: 'geojson',
+  data: source.data,
+  cluster: true,
+  clusterMaxZoom: 14, // Max zoom to cluster points on
+  clusterRadius: 100, // Radius of each cluster when clustering points
+  clusterProperties: {
+    organizationsCount: ['+', ['get', 'organizationsCount']],
+    projectsCount: ['+', ['get', 'projectsCount']],
+  },
+});
+
+const sharedLayout: SymbolLayer['layout'] = {
+  'icon-image': 'rect',
+  'icon-allow-overlap': true,
+  'icon-ignore-placement': true,
+  'text-size': 12,
+  'text-anchor': 'top',
+  'text-allow-overlap': true,
+  'text-ignore-placement': true,
+};
+
+const organizationsStyle: { layout: SymbolLayer['layout']; paint: SymbolLayer['paint'] } = {
+  layout: {
+    ...sharedLayout,
+    'icon-size': ['step', ['get', 'organizationsCount'], 1, 1, 1.5, 10, 2, 50, 2.5, 100, 2.5],
```

Comment:
Have you tested if the organisation and project rectangles can have a different height? I don't see any example on the map.

Reply:
<img width="189" alt="image" src="https://github.com/Orcasa-Platform/orcasa/assets/9701591/f2fb5154-1bc6-4ad2-bebf-4c585c439bc6">
You can see an example here the 23-24 is bigger than the 1-1

Reply:
Do you have an example where the red rectangle is smaller or bigger than the blue?

Reply:
Not really, you will have to tweak the values on the code